apps/portal/app/routes/map.tsx

```
@@ -0,0 +1,29 @@
+    }
+
+    const data = await req.json()
+
+    return data.docs
+  }
+  catch (error) {
+    console.error(`Error fetching challenges`, error)
+  }
+}
+
+export default function Map() {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)